test/standalone/map_xml_test.cpp

```diff
@@ -156,6 +156,20 @@ TEST_CASE("map xml I/O") {
         }
     } // END SECTION
 
+    SECTION("duplicate styles only throw in strict mode") {
+        std::vector<bfs::path> broken_maps;
+        add_xml_files("test/data/broken_maps", broken_maps);
```

Comment:
:normal dk
(that's vim command to delete this line and the one above ;))

Reply:
nice catch! thank you. fixed in d60be13ed